### PR TITLE
Added OTP fields to CRD

### LINKED ISSUE #532: Add OTP fields to the realm resource
### Description

The operator supports configuring the realm, but the KeycloakRealm resource is missing the OTP fields from the [RealmRepresentation](https://www.keycloak.org/docs-api/17.0/rest-api/index.html#_realmrepresentation) model, so you can't configure Keycloak to use them.

### Discussion

_No response_

### Motivation

You can't currently configure the OTP fields via the CRD

### Details

_No response_

### PR DESCRIPTION
**Pull request by stustison**
## Related GH Issue
Closes #532 

## Additional Information
Adds the OTP fields to the CRD

## Checklist:
- [ ] Automated Tests
- [ ] [Documentation](https://github.com/keycloak/keycloak-documentation) changes if necessary
- [ ] (If required) Discussed on [Keycloak DEV](https://groups.google.com/forum/#!forum/keycloak-dev)

### CODE REVIEW

File: pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go [hunk 1]
```diff
@@ -195,6 +195,34 @@ type KeycloakAPIRealm struct {
 	// User Managed Access Allowed
 	// +optional
 	UserManagedAccessAllowed *bool `json:"userManagedAccessAllowed,omitempty"`
+
+    // OTP Policy Algorithm
+    // +optional
+	OtpPolicyAlgorithm string `json:"otpPolicyAlgorithm,omitempty"`
```
Comment (vmuzikar) [inline pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go:201]: Can you please fix the indentation here?

File: pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go [hunk 2]
```diff
@@ -195,6 +195,34 @@ type KeycloakAPIRealm struct {
...
+
+	// OTP Policy Digits
```
Comment (vmuzikar) [inline pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go:203]: Can you please fix the indentation here?

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@stustison Thank you for your contribution!

Closed and reopened the PR to fix the weird CI.

File: pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go (hunk 1 above)
Comment (stustison) [reply pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go:201]: Should be fixed.

File: pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go (hunk 2 above)
Comment (stustison) [reply pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go:203]: Should be fixed.

**Review by vmuzikar — APPROVED** [review]
@stustison LGTM, thank you!

**Pull request merged**